README.md

```diff
@@ -6,14 +6,18 @@
 - a Linkurious Enterprise 3.0.4 or higher with `admin` access rights ([getting started guide](https://doc.linkurio.us/admin-manual/latest/getting-started/))
 
 ## Topics:
-1. [Importing the data in Neo4j](#1-importing-the-data-in-neo4j)
-2. [Adding the datasource in Linkurious Enterprise](#2-adding-the-datasource-in-linkurious-enterprise)
-3. [Setting default styles](#3-setting-default-styles)
-4. [Setting default captions](#4-setting-default-captions)
-5. [Setting queries](#5-setting-queries)
-6. [Setting alerts](#6-setting-alerts)
-7. [Setting custom actions](#7-setting-custom-actions)
-8. [Setting the plugin](#8-setting-the-plugin)
+- [AML Dataset implementation guide](#aml-dataset-implementation-guide)
```

Comment:
Be careful because the toc has changed due the plugin. Add some `<!-- omit in toc -->` in the first 3 chapters to remove them from here.

Also add the `<!-- auto generated with https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one -->` comment before the topics so that we know how to maintain it